### PR TITLE
SSCSSI-63: replace with v2 calls

### PR DESCRIPTION
**Pull request by sanjayparekh-hmcts**
### Jira link

https://tools.hmcts.net/jira/browse/SSCSSI-63

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by srochani — DISMISSED** [review]

**Review by nitinprabhuhmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/RoboticsCallbackHandler.java [hunk 1]
```diff
@@ -93,47 +98,68 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             log.info("Is case valid to send to robotics {} for case id {}", isCaseValidToSendToRobotics, callback.getCaseDetails().getId());
 
             if (isCaseValidToSendToRobotics) {
-                updateRpc(callback);
+                updateRpc(callback.getCaseDetails().getCaseData());
                 roboticsService.sendCaseToRobotics(callback.getCaseDetails());
```
Comment (SomeshRA) [inline src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/RoboticsCallbackHandler.java:102]: Should robotic call be in the mutator of - updateCcdCaseService.updateCaseV2

**Review by srochani — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/RoboticsCallbackHandler.java (hunk 1 above)
Comment (SomeshRA) [inline src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/RoboticsCallbackHandler.java:102]: Should the robotic call be in the mutator of - updateCcdCaseService.updateCaseV2?

Comment (sanjayparekh-hmcts) [reply src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/RoboticsCallbackHandler.java:102]: talked it through with Somesh - verified this setup is correct

**Review by SomeshRA — APPROVED** [review]

**Review by SomeshRA — APPROVED** [review]
LGTM

**Review by srochani — APPROVED** [review]

**Pull request merged**